department-of-veterans-affairs/va.gov-cms
# VACMS-1996: Sort content types alphabetically in admin menu.

## Description

See #1996 . 

## Testing done

Manual

## Screenshots

![Content___VA_gov_CMS](https://user-images.githubusercontent.com/643678/84101084-375d1300-a9db-11ea-8bc8-3ba5c83eab29.jpg)

## QA steps

As any user, go to "Admin > Add Content" using the admin menu
Content types should be sorted alphabetically by label.
